addon/components/frost-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Component definition for the frost-error component
+ */
+import layout from '../templates/components/frost-error'
+import Component from './frost-component'
+import {PropTypes} from 'ember-prop-types'
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  layout,
+  tagName: 'div',
+
+  // == PropTypes =============================================================
+
+  /**
+   * Properties for this component. Options are expected to be (potentially)
+   * passed in to the component. State properties are *not* expected to be
+   * passed in/overwritten.
+   */
+  propTypes: {
+    // options
+    description: PropTypes.string,
+    suggestion: PropTypes.string,
+    errorCode: PropTypes.number,
+    errorTitle: PropTypes.string,
+    errorMessage: PropTypes.string,
+    errorDetails: PropTypes.string,
+    showDetails: PropTypes.bool
```

Comment:
In this particular case I don't see a lot of utility in holding the expand/collapse state outside of component (retain the expansion when the user refreshes an error page?) - so I would suggest making `showDetails` internal to reduce the footprint of the interface for a consumer.